tests/e2e/default/navigation/more-options-menu/online-user.wdio-spec.js

```diff
@@ -0,0 +1,81 @@
+const commonPage = require('../../../../page-objects/default/common/common.wdio.page');
+const loginPage = require('../../../../page-objects/default/login/login.wdio.page');
+const contactPage = require('../../../../page-objects/default/contacts/contacts.wdio.page');
+const reportPage = require('../../../../page-objects/default/reports/reports.wdio.page');
+const seeder = require('./seeder');
+
+describe('Export tests', async () => {
```

Comment:
What about `More Options Menu - Online User`? 
Because it's testing more than just if the "Export" is enabled or disabled.